test/puppetlabs/trapperkeeper/core_test.clj

```diff
@@ -45,12 +45,13 @@
                broken-service])))
         (is (logged? #"Error during app buildup!" :error)
             "App buildup error message not logged")))
-    (is (thrown-with-msg?
-         RuntimeException #"Service does not define function 'foo'"
-         (macroexpand '(puppetlabs.trapperkeeper.services/service
-                         puppetlabs.trapperkeeper.core-test/FooService
-                         []
-                         (init [this context] context)))))))
+    (try (macroexpand '(puppetlabs.trapperkeeper.services/service
```

Comment:
@speedofdark didn't you run into this when you were upgrading tk? I thought there was some discussion about it, but I couldn't remember the specifics....

Reply:
The errors I'm seeing in Travis don't really look familiar. IS this the wrapping an exception vs. not thing we ran into?

Reply:
I thought it was related to that (though I might be wrong) do you remember what the outcome of that conversation was?

Reply:
Sorry, the errors in travis were because the .travis.yml configuration was out of date. I've pushed an update to fix that.

Reply:
(we ended up unwrapping the exception so it would be able to deal with the raw one vs. the wrapped one, if I recall correctly.)